### ISSUE TITLE
log spam

### ISSUE THREAD

**Issue by PeterPablo** [mikebrady/shairport-sync#273]
Hey Mike!

it's been a while -- this is primarily due to me being extremely happy with commit https://github.com/mikebrady/shairport-sync/commit/cb240896a3ea0759e4c0a5724e7f6894c0ed91d8. I used that for the last couple of weeks and was simply happy. (In case you ever get asked again about network link quality, then please ask for `ifconfig` and then divide the dropped packets by the received packages (Rx) -- my observed dopouts vanished as soon as reached a ratio below 1% and I used the above mentioned commit.)

Now I upgraded to b64c0f5c5ca0da82fcacd59b9f74b842b2309edf and things so far mostly work great. I though just had a radio station running from within Kodi and then wanted to initiate playback to shairport sync from my cell phone. My mistake was to not stop the radio station first...
My `syslog` and `daemon.log` got spammed with 400 MB of:
```Apr 19 22:28:50 rpi2 shairport-sync[5277]: failed to accept connection: Too many open files```

It isn't urgent, but you might want to restrict yourself to only log such a message once?

Kind regards from your happy user!
  Peter

**Comment by mikebrady** [comment]
Hi Peter. I am very glad that Shairport Sync is working well for you!

I have not come across that error message before. Can you give a sure-fire sequence of operations that will cause it? It looks like some interplay between it an the Kodi system... Then I'd be happy to look at it...

**Comment by mikebrady** [comment]
Hi Peter. Is this still a problem?

**Comment by PeterPablo** [comment]
I was unable to reproduce it as of now. Please close this issue (sent from
mobile).

**Comment by mikebrady** [comment]
Okay, thanks. Please feel free to reopen it if there are any developments.

**Comment by teake** [comment]
I've had this happen too, with `syslog` and `daemon.log` each about 7.5 GB big. Unfortunately I had to clear both files to free up some space on the SD card, so can't provide much debug information. I'm on 2.8.4.0, and had a Plex server running in the background.

**Comment by mikebrady** [comment]
Thanks. If you happen to get some more of it, it would be great to see it.

**Comment by teake** [comment]
I've had another encounter. I was playing some files through Mopidy beforehand, but restarted that service before attempting to play something through shairport sync. I've attached my log files in case you want to have a look:

[syslog.zip](https://github.com/mikebrady/shairport-sync/files/362224/syslog.zip)
[daemon.log.zip](https://github.com/mikebrady/shairport-sync/files/362225/daemon.log.zip)

I also noticed that shairport sync logs the following every 3 seconds when the network is unavailable: `Error sendto-ing to timing socket: Network is unreachable`. I disable the wireless at night, so this also quickly fills up my logs (although not as quick as the other message).
